sdk/mixedreality/mixedreality-authentication/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const SDK_VERSION: string = "1.0.0-prerelease.1";
```

Comment:
This doesn't match what's in package.json. Assuming you want the `beta` tag.
```suggestion
export const SDK_VERSION: string = "1.0.0-beta.1";
```

Reply:
Yea. I just changed to use `beta` since that is what we've been told to do in all the other languages. I obviously missed this one. Thanks!